packages/api-bindings/src/graphql/client.graphql

```diff
@@ -12,16 +13,19 @@ extend type Profile {
 extend type Post {
   hasOptimisticCollectedByMe: Boolean!
   isOptimisticMirroredByMe: Boolean!
+  collectState: CollectState!
```

Comment:
Question: any reason to not have adopted the same strategy you used with `followPolicy: FollowPolicy`?

It did seems pretty analogous scenario given the number of details some collect policy requires compared to others?